unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FeatureController/Test/PluginSystemShould.cs

```diff
@@ -8,7 +8,20 @@
 
 public class PluginSystemShould : IntegrationTestSuite
 {
+    private GameObject _main;
```

Comment:
```suggestion
    private GameObject main;
```
Remember that we don't use underscore for private variables.